src/openforms/formio/components/vanilla.py

```diff
@@ -150,9 +142,28 @@ def build_serializer_field(
 
 
 @register("time")
-class Time(BasePlugin):
+class Time(BasePlugin[Component]):
     formatter = TimeFormatter
 
+    def build_serializer_field(
+        self, component: Component
+    ) -> serializers.TimeField | serializers.ListField:
+        multiple = component.get("multiple", False)
+        validate = component.get("validate", {})
+        required = validate.get("required", False)
+
+        validators = []
+        if min_time := validate.get("minTime"):
+            validators.append(MinValueValidator(time.fromisoformat(min_time)))
+        if max_time := validate.get("maxTime"):
+            validators.append(MaxValueValidator(time.fromisoformat(max_time)))
```

Comment:
I don't really see how the `minTime: 20:00` and `maxTime: 4:00` are handled here - it doesn't look like it'll be valid for a value like `02:00` - can you add some unit tests too? Rather than only the end-to-end tests.

Reply:
I took care of it :)